### DISCUSSION TITLE
hotkeys to enable/disable set plugin/s

### DISCUSSION THREAD

**Discussion by Erutpur** [runelite/runelite#18174] [Ideas]
Hello, dear Runelite Team,
Wouldn't it be cool to have the possibility to just press one hotkey to enable or disable a plugin at any given time?
There would be no need to open the search bar if you have set up the hotkey beforehand. If you have plugins that you need to be enabled/disabled for different activities, it would be such a cool feature to just use a hotkey. It's more efficient, time saving and definitely more convenient.

**Comment by DominickCobb-rs** [comment]
You might be interested in [plugin presets](https://runelite.net/plugin-hub/show/plugin-presets)
